base/sink/FairRootFileSink.cxx

```diff
@@ -107,8 +107,7 @@ Bool_t FairRootFileSink::InitSink()
 
     // FairRun* fRun = FairRun::Instance();
     /**Check if a simulation run!*/
-    fOutFolder = gROOT->GetRootFolder()->AddFolder(
-        Form("%s_%d", FairRootManager::GetFolderName(), FairRootManager::Instance()->GetInstanceId()), "Main Folder");
+    fOutFolder = gROOT->GetRootFolder()->AddFolder(Form("%s", FairRootManager::GetFolderName()), "Main Folder");
```

Comment:
From what I understand, this changes an API towards the user?
The generated Folders (and below the trees) now have different names?

Could you add a small entry to the "Breaking Changes" section in the changelog please?

Reply:
Okay, I start to get it, after reading the docs on TFolder.

Maybe this code should be locked with a mutex, and check wether the Folder already exists and only create it, if it does not yet exist?

Well, still I feel like I am missing something.

Reply:
The TFolder may (and in FairRoot is) be used to automatically create the tree structure. We used to create folders with different names for the thread workers, but then created trees are not read properly when TChain'ed.